packages/strapi/lib/middlewares/gzip/index.js

```diff
@@ -12,7 +12,8 @@ module.exports = strapi => {
      */
 
     initialize() {
-      strapi.app.use(compress());
```

Comment:
Hey, can you instead get the rest of options from the gzip settings like follows:


```js
const { enableBrotli = false, ...options } = strapi.config.middleware.settings.gzip;

strapi.app.use(compress(options));
```

Reply:
Done